src/signature_pad.ts

```diff
@@ -248,15 +243,18 @@ export default class SignaturePad extends SignatureEventTarget {
     // Prevent scrolling.
     event.preventDefault();
 
-    const touch = event.targetTouches[0];
-    this._strokeMoveUpdate(touch);
+    if (this._drawningStroke) {
+      const touch = event.targetTouches[0];
+      this._strokeMoveUpdate(touch);
+    }
   };
 
   private _handleTouchEnd = (event: TouchEvent): void => {
     const wasCanvasTouched = event.target === this.canvas;
     if (wasCanvasTouched) {
       event.preventDefault();
 
+      this._drawningStroke = false;
```

Comment:
Do we need to worry about this ending when a touch is still occuring?

For example:

1. First finger touch (`_handleTouchStart` called)
1. Second finger touch (`_handleTouchStart` called)
1. Second finger removed (`_handleTouchEnd` called)
1. First finger moves but nothing happens because `_drawningStroke` is set to false when the second finger was removed.